utils.py

```diff
@@ -159,3 +159,23 @@ def _compare_endpoints(endpoint1, endpoint2):
     elif 'out_vlan' in endpoint1 or 'in_vlan' in endpoint2:
         return False
     return True
+
+
+def convert_vlan(value):
+    """Auxiliar function to calculate dl_vlan"""
+    if isinstance(value, int):
+        return value, 4095
+    value, mask = map(int, value.split('/'))
```

Comment:
@Alopalao, thanks for your review and comments. 
I removed the cases of `untagged` and `any` due to @viniarck suggestion, I quote his comments:
"Since we're not changing the API we shouldn't be accepting "dl_vlan": "untagged" in the request payload body"

Reply:
@Alopalao, let's adapt mef_eline to either send a `dl_vlan` within [1, 4095] (maybe having a deterministic value here is fine since the ranmdoness has already been covered in e2e tests) when it's `"any"` or without `dl_vlan` when it's port-based or when it's untagged. The rationale being, that since sdntrace_cp is performing a packet tracing lookup let's maintain the inputs what's valid for a packet. That should also simplify for vlan range in the future and keep the boundaries clear, while keepin sdntrace_cp inputs simple to maintain without many mappings/translations. In fact, for vlan range for instance, mef_eline is the only know how the flow entries are partitioned how it knows best how to better/properly to do this.

Reply:
LGTM

Reply:
Closing this since a PR have been created for this.